modules/gapi/cmake/DownloadADE.cmake

```diff
@@ -17,7 +23,8 @@ if (NOT res)
     return()
 endif()
 
-set(ADE_root "${ade_src_dir}/${ade_subdir}/sources/ade")
+set(ADE_root "${ade_src_dir}/${ade_${OPENCV_DOWNLOAD_HOST}_subdir}/sources/ade")
```

Comment:
Can we use commit id as what is done for Tengine to avoid changing ADE_root?

Reply:
Packages downloaded from the release page of Gitlab-style mirrors adds a 'v' to the extracted directory name. For example "oneTBB-2020.2" is the extracted directory name of the package from Github, "tbb-v2020.2-${commid_id}" is the one of the package from Gitcode.

Reply:
Done.